day1/part1.py

```diff
@@ -1,13 +1,9 @@
-def find_first_last_digit(line: str) -> str:
-    for character in line:
+def find_digit(line: str, directio: int) -> str:
```

Comment:
xd

This is bad practice, do not adjust the class to bend to the input you place in just adjust the input.
Make it so it takes just one argument and you send in line and line[::-1].